package.json

```diff
@@ -281,5 +281,8 @@
     "tools/@*/*",
     "apps/*"
   ],
+  "cli": {
```

Comment:
Why not using the [config](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#config) object from package.json standard spec?